pgc_ortho.py

```diff
@@ -104,7 +108,7 @@ def main():
         # by default, the parent directory of the dst dir is used for saving slurm logs
         if args.slurm_log_dir == None:
             slurm_log_dir = os.path.abspath(os.path.join(dstdir, os.pardir))
-            print("slurm log dir: {}".format(slurm_log_dir))
+            logger.info("slurm log dir: {}".format(slurm_log_dir))
```

Comment:
Logger used before any handlers are defined.

Reply:
I see a few more instances of this.  But I'll refrain from commenting to keep the conversation cleaner.

Reply:
I pushed an update for all the logging order of operations so there should not be references to logging before the handler is set up. However, in `mosaic.py` there are a few previously existing `logger` references in `main()` but the only place where the file and stream handlers are set up is within `run_mosaic()`. I didn't change those logger references since they predate this PR and they should be picked up by the default behavior of the logger.